client-ts/Microsoft.AspNetCore.SignalR.Client.TS/MessagePackHubProtocol.ts

```diff
@@ -0,0 +1,117 @@
+import { IHubProtocol, MessageType, HubMessage, InvocationMessage, ResultMessage, CompletionMessage } from "./IHubProtocol";
+import { BinaryMessageFormat } from "./Formatters"
+
+var msgpack = require("msgpack-lite");
```

Comment:
They added typings a day after I wrote this code. Switching to typings will make this ugliness go away.